README.md

```diff
@@ -375,6 +375,16 @@ You can run Dapr's sidecar only (`daprd`) by omitting the application's command
 $ dapr run --app-id myapp --port 3005 --grpc-port 50001
 ```
 
+### Generate shell completion scripts
```

Comment:
Is there a way to support completion in windows? otherwise, we may need to add statement.

Reply:
@youngbupark I found that the underlying library, Cobra, supports autocompletion for powershell, so I'll try to update the code to support powershell as well. I'll let you know once I finish the work.

Reply:
I updated the autocompletion feature for windows powershell as well as bash and zsh. Please let me know if there is anything I could improve.